ansible/roles/collectory/tasks/vm-tasks.yml

```diff
@@ -18,21 +18,69 @@
     - db
     - collectory
 
+
+#deploy as systemd service
+- name: add collectory jar and setup systemd service
+  include_role:
+    name: exec-jar
+  vars:
+    service_name: 'collectory'
+    use_openjdk17: "{{ collectory_version | string is version('5.2.0', '>') }}"
+    log_config_filename: "{{ logback_config_filename }}"
+    java_headless: True
+  tags:
+    - deploy
+    - service
+    - collectory
+  when: exec_jar | default(false) | bool
+
+#todo: check if this is needed
```

Comment:
Answering the question, this check (directory exists) is not required because it is performed by the nginx_vhost role.

Reply:
removed